python/deploy.py

```diff
@@ -116,8 +140,20 @@ async def deploy_resources(self, resource_paths: list, tenant_id: str):
         pass
 
     try:
-        if os.environ["CAMUNDA_CLUSTER_HOST"] is not None and os.environ["CAMUNDA_CLUSTER_HOST"] != "":
-            deploy.set_cluster_host(os.environ["CAMUNDA_CLUSTER_HOST"])
+        if os.environ["OAUTH_TOKEN_URL"] is not None and os.environ["OAUTH_TOKEN_URL"] != "":
```

Comment:
This doesn't match the example (inconsistent `OAUTH` vs `OAUTH2`
```suggestion
        if os.environ["OAUTH2_TOKEN_URL"] is not None and os.environ["OAUTH2_TOKEN_URL"] != "":
```